src/pages/components/multiselect.mdx

```diff
@@ -6,6 +6,12 @@ description: >
 package: '@zendeskgarden/react-dropdowns'
 propSheets:
   - dropdowns/src/elements/Multiselect/Multiselect.tsx
+  - dropdowns/src/elements/Dropdown/Dropdown.tsx
+  - dropdowns/src/elements/Fields/Field.tsx
+  - dropdowns/src/elements/Fields/Hint.tsx
+  - dropdowns/src/elements/Fields/Label.tsx
+  - dropdowns/src/elements/Menu/Items/Item.tsx
+  - tags/src/elements/Tag.tsx
```

Comment:
I'm not sure `Tag` should be included here. This is located in a different package.

Reply:
+1 Especially since it auto-populates the "Configuration" import string. There should be verbiage in this page that links off to the Tags documentation.

Reply:
Phew! Great callout. That solves that